mentor-booking.js

```diff
@@ -25,10 +31,17 @@ const bookings = [];
  * Devuelve una lista de mentores o mentoras presentes en la lista de usuarios.
  * @return {Array} Lista de mentores o mentoras
  */
+
 const getMentors = () => {
-  // TODO 1
-};
 
+  const mentorsList = [];
```

Comment:
We can also use a `.filter` iterator here, WDYT?
```
return users.filter(user => user.rol === "MENTOR")
```

Reply:
Done